admin/src/views/Overview.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
Hier ist viel zu übersetzen.

Reply:
die übersetzungen würde ich gern in einem extra arbeitsschritt machen. das ist momentan zu stressig dies zu beachten. und würde mich im basteln extrem aufhalten und vor allem verwirren .. weil ich die deutschen strings sehen muss um den überblick zu behalten .. 
zu viel ändert sich noch.